lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -157,6 +160,13 @@ def configure(conf)
       if (@rename_time || @rename_time_if_not_exist) && @use_undefined && !@keep_fields.key?(@src_time_name)
         raise Fluent::ConfigError, "Field [#{@src_time_name}] must be listed in default_keep_fields or extra_keep_fields"
       end
+      @event_trans = @event_trans.to_sym
+      allowed_trans_options = [:none, :eventrouter]
+      if allowed_trans_options.include?(@event_trans)
+        log.info("Set event transformation: "+@event_trans.to_s)
```

Comment:
running fluentd with `trace` level logging will dump all of the config.
If you want to check the config.  At any rate, if you use an `enum` for `event_trans`, all of this code goes away.